docs/EVG.md

```diff
@@ -760,6 +755,18 @@ Before issuing an EV Certificate, the CA MUST ensure that all Subject organizati
 
 As a general rule, the CA is responsible for taking all verification steps reasonably necessary to satisfy each of the Verification Requirements set forth in the subsections below.  The Acceptable Methods of Verification set forth in each of Sections 11.2 through 11.14 (which usually include alternatives) are considered to be the minimum acceptable level of verification required of the CA.  In all cases, however, the CA is responsible for taking any additional verification steps that may be reasonably necessary under the circumstances to satisfy the applicable Verification Requirement.
 
+### 11.1.3.  Disclosure of Verification Sources
+
+Effective as of 1 October 2020, prior to the use of an Incorporating Agency or Registration Agency to fulfill these verification requirements, the CA MUST publicly disclose, through an appropriate and readily accessible online means, Agency Information about the Incorporating Agency or Registration Agency.
+
+This Agency Information SHALL include at least the following:
+* Sufficient information to unambiguously identify the Incorporating Agency or Registration Agency (such as a name, jurisdiction, and website); and,
+* The accepted values for the `subject:jurisdictionLocalityName` (OID: 1.3.6.1.4.1.311.60.2.1.1), `subject:jurisdictionStateOrProvinceName` (OID: 1.3.6.1.4.1.311.60.2.1.2), and `subject:jursidictionCountryName` (OID: 1.3.6.1.4.1.311.60.2.1.3) fields when a certificate is issued using information from that Incorporating Agency or Registration Agency, indicating the jurisidction(s) that the Agency is appropriate for; and,
+* The accepted or allowed form or syntax of the Registration Number used by the Incorporating Agency or Registration Agency, if known; and,
```

Comment:
Defining the accepted or allowed form or syntax is going above and beyond the intent of publishing a list of Agencies and should be removed from this ballot.

Reply:
@dougbeattie This is "if known" - it's optional.

Without this information, we're limiting ourselves to assuming that the only way we'll be able to map Incorporating Agencies / Registration Agencies is if we know all jurisdictions have 1 and exactly one such agency (which we know they don't) or we have a way to tie it to the certificate (e.g. via a new field). This is trying to explore if there's a possibility of a different option, which is using the serial number itself.

This is trying to be optional, but to call it out as important to disclose, especially as some CAs have already done so.

Reply:
""if known" leaves a lot of room for interpretation other than "optional".  Maybe a CA doesn't know because they didn't look, does that mean it's "not known"?  If you replace with "optional", then it's clear.

Is the intent of this ballot to enable mapping?  I'd suggest starting with CAs disclosing their Agencies and then take the next step to provide mapping details.  For most regions, there are 1 or only a couple of Agencies, so if someone wanted to "map" they'd have a pretty good idea of which Agency was used, even

Reply:
I attempted to tweak this in https://github.com/sleevi/cabforum-docs/pull/11/commits/74c8f85eaeb0a423c5d0395c6a53ee9786e5411d to make it clearer that it only applies if the CA itself restricts. Does that better address your concern?

Reply:
Just following up to see if this addressed @dougbeattie's concern